<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgtWindowInfo</class>
 <widget class="QWidget" name="WgtWindowInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>122</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblOptions">
        <property name="toolTip">
         <string>CMake code inserted before adding the subdirectory</string>
        </property>
        <property name="text">
         <string>Window Title</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="txtTitle"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnRemoveWindow">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../data/HyEditor.qrc">
            <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Window Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbType">
        <item>
         <property name="text">
          <string>Windowed Fixed Size</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Windowed Resizable</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full Screen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Borderless Window</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblProjectName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="WgtVectorSpinBox" name="vsbResolution" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblProjectName_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="WgtVectorSpinBox" name="vsbLocation" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WgtVectorSpinBox</class>
   <extends>QWidget</extends>
   <header>WgtVectorSpinBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
